src/config/ConfigParser.ts

```diff
@@ -474,6 +571,46 @@ export function configParseGasLimitsPrices(slice: Slice | null | undefined) {
     }
 }
 
+function configParseLimitParams(slice: Slice) {
+    const paramsLimitTag = slice.loadUint(8);
+
+    if (paramsLimitTag === 0xc3) {
+        const underload = slice.loadUintBig(32);
+        const softLimit = slice.loadUintBig(32);
+        const hardLimit = slice.loadUintBig(32);
+
+        return {
+            underload,
+            softLimit,
+            hardLimit
+        };
+    }
+    throw Error('Invalid config');
+}
+
+export type BlockLimits = ReturnType<typeof configParseBlockLimits>;
```

Comment:
This will force Ctrl+Click to direct user to lines 583-585. Also there is no place to put comments on these fields.

Please define the type with `type BlockLimits = { ... }`.